package/upgrade/Dockerfile

```diff
@@ -23,3 +23,4 @@ COPY patch_99_custom.sh /usr/local/bin
 COPY extra_manifests /usr/local/share/extra_manifests
 COPY migrations /usr/local/share/migrations
 COPY addons /usr/local/share/addons
+COPY elemental_cli /usr/local/bin/elemental_cli
```

Comment:
how do we get the elemental_cli and what's the current installed version?

Reply:
This element-cli is copy from [v0.0.14-harvester1](https://github.com/rancher/elemental-cli/releases/tag/v0.0.14-harvester1).

That related discussions are:
https://github.com/rancher/elemental-cli/issues/348
https://github.com/rancher/elemental-cli/pull/350

That could resolve the issue that failure when elemental upgrade on duplicated disk label environment.

Reply:
@Vicente-Cheng Let's not add the binary to the source but download it in the Dockerfile with RUN and curl.